### ISSUE TITLE
Replace arguments of decorator

### ISSUE THREAD

**Issue by elclanrs** [dsherret/ts-simple-ast#56]
I want to replace the arguments of an Angular `@Component` decorator. I couldn't find a `removeDecorator` and I tried `addDecorator` but I'm unsure of what `arguments?: string[]` is supposed to look like.

What is the best way to do this?

**Comment by dsherret** [comment]
Right now there's not a good way to remove decorator or their arguments. I'm slowly working on this in #47.

I updated the documentation on decorators to show an example of `arguments` ([see here](https://dsherret.github.io/ts-simple-ast/details/decorators)). Basically, you can provide an array of strings like so:

```typescript
classDeclaration.addDecorator({ name: "MyDecorator", arguments: ["5", `"some string"`] });
```

**Comment by elclanrs** [comment]
@dsherret,  thanks for updating the documentation, now I understand how to use `arguments`. Looking forward to this feature.

**Comment by dsherret** [comment]
@elclanrs this is implemented in 0.53.0.

From a decorator, get the argument you want to remove and call `.remove()` on it:

```typescript
decorator.getArguments()[2].remove();
```

Or remove an entire decorator by calling `.remove()` on it:

```typescript
decorator.remove();
```
